### PR TITLE
 Changes for pre-installed Istio and service mesh client

### PR DESCRIPTION
**Pull request by chiragnarangmulesoft**
New changes for supporting pre-installed Istio and renaming Anypoint Service Mesh client from "service-mesh" to asmctl

### CODE REVIEW

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 1]
```diff
@@ -13,20 +13,22 @@ You install, manage, and troubleshoot your Anypoint Service Mesh using the Anypo
 
 To download the Anypoint Service Mesh client, type the following command at the command prompt:
 
-`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > service-mesh && chmod +x service-mesh`
+`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
 
-The Anypoint Service Mesh client is now downloaded on your file system.
+The Anypoint Service Mesh client `asmctl` is now downloaded on your file system. You can verify using this command
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc:18]:
```suggestion
The Anypoint Service Mesh client `asmctl` is now downloaded on your file system. You can verify using this command:
```

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 2]
```diff
@@ -13,20 +13,22 @@ You install, manage, and troubleshoot your Anypoint Service Mesh using the Anypo
...
-When you invoke the installer, Istio, Service Catalog, Anypoint Service Mesh Broker, and ClusterIP Service components are installed in your Kubernetes cluster. The installer performs several steps as part of the installation, such as checking for dependencies and performing various configurations. 
+When you invoke the installer, Service Catalog, Anypoint Service Mesh Broker, and ClusterIP Service components are installed in your Kubernetes cluster. The installer performs several steps as part of the installation, such as checking for dependencies and performing various configurations. 
 
 === Install without Proxy
 To install Anypoint Service Mesh, perform the following steps:
 
 . In the location from which you downloaded your Anypoint Service Mesh client, type:
 +
-`./service-mesh install`
+`asmctl install`
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc:31]:
```suggestion
`./asmctl install`
```

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 3]
```diff
@@ -52,7 +54,7 @@ If you need to install Anypoint Service Mesh in a proxy environment, provide the
 
 . In the location from which you downloaded your Anypoint Service Mesh client, type:
 +
-`./service-mesh install --clientId=<clientId> --clientSecret=<clientSecret> --license=<license absolute or relative path> --platformUri=<Anypoint Platform Uri> --proxyHost=<host> --proxyProtocol=<protocol> --proxyPort=<port> --proxyUser=<proxy username> --proxyPassword=<proxy password>`
+`asmctl install --clientId=<clientId> --clientSecret=<clientSecret> --license=<license absolute or relative path> --platformUri=<Anypoint Platform Uri> --proxyHost=<host> --proxyProtocol=<protocol> --proxyPort=<port> --proxyUser=<proxy username> --proxyPassword=<proxy password>`
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc:57]:
```suggestion
`./asmctl install --clientId=<clientId> --clientSecret=<clientSecret> --license=<license absolute or relative path> --platformUri=<Anypoint Platform Uri> --proxyHost=<host> --proxyProtocol=<protocol> --proxyPort=<port> --proxyUser=<proxy username> --proxyPassword=<proxy password>`
```

**Review by wwarner-salesforce — APPROVED** [review]

**Comment by tbruno** [comment]
This PR is stale.

**Pull request closed without merging**